spec/rubocop/cop/style/case_condition_repeat_spec.rb

```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::CaseConditionRepeat do
+  subject(:cop) { described_class.new }
+
+  let(:offense_source) do
```

Comment:
I'd group the lets and their respective tests in `describe` blocks. Right now it's hard to see which source snippet is used with which test.